docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -122,6 +122,18 @@ It is recommended you do a [backup]({{< relref "doc/usage/backup-and-restore.en-
 If you have carried out the installation steps as described above, the binary should
 have the generic name `gitea`. Do not change this, i.e. to include the version number.
 
+### 1. Use systemd (recommended)
```

Comment:
In line with my comment below, I suggest using `### 1. Restarting gitea with systemd (recommended)`